flexget/plugins/cli/database.py

```diff
@@ -77,16 +79,21 @@ def reset_plugin(options):
             console('Unable to reset %s: %s' % (plugin, e.message))
 
 
+def backup(manager):
+    fire_event('manager.backup_db', manager)
```

Comment:
Hmm, a plugin is firing a manager event? This doesn't feel quite right to me. I think the framework may support registering this from the backup plugin, where it can run itself directly, and skip the event.